src/TMS/dev/TCA9545A.cpp

```diff
@@ -0,0 +1,40 @@
+#include <TMS/dev/TCA9545A.h>
+
+namespace TCA9545A {
+
+TCA9545A::TCA9545A(IO::I2C& i2c) : i2c(i2c){};
+
+IO::I2C::I2CStatus TCA9545A::setBus(uint8_t bus, bool toggled) {
+    IO::I2C::I2CStatus ret;
+    uint8_t val = toggled ? 1 : 0;
+    switch (bus) {
+    case 0:
+        ret = writeRegister(TCA9545A_BUS::BUS_0, val);
+        break;
+
+    case 1:
+        ret = writeRegister(TCA9545A_BUS::BUS_1, val);
+        break;
+
+    case 2:
+        ret = writeRegister(TCA9545A_BUS::BUS_2, val);
+        break;
+
+    case 3:
+        ret = writeRegister(TCA9545A_BUS::BUS_3, val);
+        break;
+    default:
+        return IO::I2C::I2CStatus::ERROR;
+    }
+
+    return ret;
```

Comment:
Just put return statements in each case, so you can get rid of all the `break`s

Reply:
Aren't breaks needed in c++ to stop a switch statement from continuing to execute? I believe if the break is removed then for example, if bus == 1 it would write to registers BUS_1, BUS_2, and BUS_3 as well as returning an I2C error.

Reply:
@AarC10 we discussed this in person and came to the conclusion that the break statements are unnecessary